charts/helm-operator/values.yaml

```diff
@@ -15,7 +15,7 @@ crd:
   listKind: HelmReleaseList
 
 image:
-  repository: quay.io/coreosinc/helm-operator
+  repository: "mustbereplaced"
   tag: latest
```

Comment:
PR looks good. 
Could `quay.io/openshift/origin-metering-helm-operator` be used as the value here?

Reply:
This chart doesn't have any default value that makes sense. We override this always anyways, but this ensures if we didn't, it would be really obvious.

Reply:
Good call/check